stripe/src/test/java/com/stripe/android/net/StripeApiHandlerTest.java

```diff
@@ -77,7 +79,7 @@ public void getHeaders_containsPropertyMapValues() {
         String userAgentRawString = headerMap.get("X-Stripe-Client-User-Agent");
         try {
             JSONObject mapObject = new JSONObject(userAgentRawString);
-            assertEquals("3.5.0", mapObject.getString("bindings.version"));
+            assertEquals(BuildConfig.VERSION_NAME, mapObject.getString("bindings.version"));
```

Comment:
This references the actual version name ("2.1.0") in the `gradle.properties` file, which is what is used to give it a published name.